doc/OnlineDocs/source/conf.py

```diff
@@ -0,0 +1,62 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Prescient'
+copyright = '2021, Prescient Developers'
+author = 'Prescient Developers'
+
+# The full version, including alpha/beta/rc tags
+release = '1.0'

```

Comment:
We should probably call this `2.0`, to match the [current version in setup.py](https://github.com/grid-parity-exchange/Prescient/blob/530a5d7a72e9b9281c11d32ce1ce3d58653e7e87/setup.py#L26).
```suggestion
release = '2.0'

```

Reply:
I noticed on the github project page it lists the latest version as 2.0.1. So bump to 2.0.2?

Reply:
That would probably be best -- I would just like this to be in lockstep with whatever is the `version` in setup.py.